unstract/tool-registry/src/unstract/tool_registry/public_tools.json

```diff
@@ -5,7 +5,7 @@
             "schemaVersion": "0.0.1",
             "displayName": "File Classifier",
             "functionName": "classify",
-            "toolVersion": "0.0.17",
+            "toolVersion": "0.0.19",
```

Comment:
Why are we bumping the version to 0.0.19 instead of 0.0.18 ?

Reply:
Resolving it. Just now checked the docker hub. 0.0.18 was missed to update here.